ansible/roles/riakcs/configure/templates/riak.conf.j2

```diff
@@ -453,7 +460,12 @@ riak_control.auth.mode = off
 ## 
 ## Acceptable values:
 ##   - an integer
+{% if _riak_backend == "leveldb-bad-config" %}
 leveldb.maximum_memory.percent = 70
+{% else %}
+## Overridden by total_leveldb_mem_percent in advanced.config
+#leveldb.maximum_memory.percent = 70
```

Comment:
does this also apply for if the backend is `bitcask`? I see this setting in the leveldb advanced config with a value of 70 but in the bitcask advanced config the setting has a value of 30. I assume that's correct but just wanted to check.

Reply:
Yes, my understanding is that if `total_leveldb_mem_percent` is present in advanced.config then this setting in riak.conf is ignored.